### ISSUE TITLE
Support run-time replacement of StationDB implementation

### ISSUE THREAD

**Issue by ohanssen** [PolaricServer/aprsd#28]
In-memory search is fast, no problem, but saving/restore of the data can take some time and the data uses some memory. The idea is that if the Database plugin is active, data could be stored in the SQL database.

**Comment by ohanssen** [comment]
This is implemented. Performance is an issue. If we use APRS data from the whole world there is about 60k items. This takes some milliseconds to search through. So I tried to move this to a database with GIST indexing, etc.. It helps significantly when we search a relatively small number of items, but getting a larger number of items from the database takes more time. I replaced serialization with FST, I used a LRU cache, and used Unix sockets for communication with the database. Then the performance is better than in-memory searching, at least when the number of searched items is less than 1000. At least on my computer :)
